storage_delegation/main.tf

```diff
@@ -6,16 +13,50 @@ locals {
     "eu-de"    = "eu-de.dataplatform.cloud.ibm.com",
     "jp-tok"   = "jp-tok.dataplatform.cloud.ibm.com"
   }
-  dataplatform_ui = local.dataplatform_ui_mapping[local.location]
+  create_access_policy_kms    = !var.skip_iam_authorization_policy
+  dataplatform_ui             = local.dataplatform_ui_mapping[local.location]
+  kms_service                 = module.crn_parser_kms_key.service_name
+  kms_account_id              = module.crn_parser_kms_key.account_id
+  kms_key_id                  = module.crn_parser_kms_key.resource
+  target_resource_instance_id = module.crn_parser_kms_key.service_instance
 }
 
 resource "ibm_iam_authorization_policy" "cos_s2s_keyprotect" {
+  count                       = local.create_access_policy_kms ? 1 : 0
```

Comment:
same as above ...We can directly use `var.skip_iam_authorization_policy`

Reply:
This has been implemented.